modules/user/user.service.js

```diff
@@ -107,6 +108,10 @@ class UserService {
 
     const user = await User.findById(id).lean();
 
+    if (!user) {
```

Comment:
Please, add a test that checks that if id of the unexisting user is provided proper error message is returned

Reply:
solved